reservacionVuelo/App.js

```diff
@@ -1,19 +1,30 @@
 import React, {Component} from 'react';
-import { SignUp } from './Screens/SignUp';
+import {View, StyleSheet} from 'react-native';
+import {Login} from './Screens/Login';
+import {SignUp} from './Screens/SignUp';
 
 class App extends Component {
   constructor(props) {
     super(props);
+    this.name = 'App';
     this.state = {
       respuesta: 'si',
```

Comment:
I would recommend that instead of "respuesta", you put "answer", because all your other code is written in the English language.

Reply:
It is actually leftover code that shouldn't be there. I didn't notice that.

Thanks.